app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt

```diff
@@ -44,5 +52,10 @@ class MainActivity : ComponentActivity() {
                 }
             }
         }
+        WindowCompat.setDecorFitsSystemWindows(window, false)
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.R) {
+            window.addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN)
```

Comment:
@yugeshjain can we add a comment here why this was needed ?

Reply:
Done